src/ds/d400/d400-device.cpp

```diff
@@ -569,7 +569,7 @@ namespace librealsense
         }
         set_hw_monitor_for_auto_calib(_hw_monitor);
 
-        _ds_device_common = std::make_shared<ds_device_common>(this, _hw_monitor);
+        _ds_device_common = std::make_shared<ds_device_common>(this, _hw_monitor, (_pid == ds::RS457_PID)? true : false);
```

Comment:
I see on this common class we have another place where we distinguish between D400 and d500
` if( auto dev = dynamic_cast< d400_device * >( _owner ) )
            return dev->get_raw_depth_sensor();`

So we can see if it's D457 by reading the camera name instead of passing this bool.

What do you think? which way is nicer?

Reply:
will check it.